Dockerfile

```diff
@@ -1,13 +1,92 @@
-ARG DOCKER_ORG=usdotfhwastoldev
-ARG DOCKER_TAG=develop
-FROM ${DOCKER_ORG}/carma-base:${DOCKER_TAG} as base_image
-SHELL ["/bin/bash", "-c"]
+FROM ros:humble-ros-base-jammy as base-install
```

Comment:
- This so that we don't have unnecessary dependency for ros1_bridge as almost all dependencies in carma-base are unnecessary for ros1_bridge. 
For example, ros1_bridge created by this approach is only 2.59 GB while carma-base:develop-humble itself is at least 4.22GB
- I was also having a trouble resolving catkin issues on develop-humble

Reply:
Ah gotcha, good point!